### PR TITLE
Add naxos importer scripts

### PR DESCRIPTION
**Pull request by loujine**
Importers for naxosmusiclibrary.com (old version of the website, still available) and nml3.naxosmusiclibrary.com (new version)

### CODE REVIEW

**Comment by loujine** [comment]
I will look at the linter errors tomorrow.
I guess I should not update the README directly but use the generate_readme script?

**Comment by murdos** [comment]
> I guess I should not update the README directly but use the generate_readme script?

Exact :-)

**Comment by loujine** [comment]
lint errors fixed + README generated by the tool script

File: naxos_library_importer.user.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+// @run-at       document-end
+// ==/UserScript==
+
+// seems that $ is predefined but does not work
+$ = jQuery;
+
+var url = document.URL.split('.');
+url.splice(0, 1, 'https://www');
+const editNote = `
+    Imported from ${url}
```
Comment (murdos) [inline naxos_library_importer.user.js:27]: URL was incorrect during my test, I got `https://www,com/catalogue/item,asp?cid=ALPHA581`

**Comment by murdos** [comment]
@loujine : do you have time to take my remarks into account?

**Comment by loujine** [comment]
@murdos: sorry, I'm just taking a look now, will push a new version shortly

File: naxos_library_importer.user.js (hunk 1 above)
Comment (loujine) [reply naxos_library_importer.user.js:26]: I don't quite understand how my code was supposed to work... :)
Anyway I fixed it

**Comment by loujine** [comment]
Fixed the issues you mentioned, cleaned up the code and ran with eslint. It should be good to merge now

**Comment by loujine** [comment]
Thanks!

**Pull request merged**